### PR TITLE
Update doc for SF4: faker-providers and alice-processors

### PR DESCRIPTION
**Pull request by RomulusED69**
Related to [issue](https://github.com/hautelook/AliceBundle/issues/406)

### CODE REVIEW

File: doc/alice-processors.md [hunk 1]
```diff
@@ -4,7 +4,16 @@ Given you declared a processor `App\DataFixtures\Processor\UserProcessor`, you h
 service with the tag `fidry_alice_data_fixtures.processor` to register it:
 
 ```yaml
-# app/config/services.yml
+# config/services_dev.yml in SF4
+
+services:
+    App\DataFixtures\Processor\UserProcessor:
+        class: App\DataFixtures\Processor\UserProcessor
+        tags:
```
Comment (theofidry) [inline doc/alice-processors.md]: to double check but I think you don't even need that tag with the autoconfigure enabled, same for faker providers, cf. https://github.com/theofidry/AliceDataFixtures/pull/65/files

Comment (RomulusED69) [reply doc/alice-processors.md]: In my project, I tried without tag and faker doesn't work. I not tested with processor

**Comment by RomulusED69** [comment]
Yes and it doesn't work

File: doc/alice-processors.md (hunk 1 above)
Comment (greg0ire) [reply doc/alice-processors.md]: Can you please check whether the piece of code in my PR is executed on your machine? You might have to set `autoconfigure` to `true` for that service I think

**Pull request closed without merging**